pkgs/by-name/ga/garmindb/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "garmindb";
+  version = "3.6.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tcgoetz";
+    repo = "garmindb";
+    tag = "v${version}";
+    hash = "sha256-JAUDAYf9CH/BxwV88ziF5Zy+3ibcbieEfHrZpHSU8m0=";
+  };
+
+  build-system = [
+    python3Packages.setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    sqlalchemy
+    python-dateutil
+    cached-property
+    tqdm
+    garth
+    fitfile
+    tcxfile
+    idbutils
+    tornado
+  ];
+
+  pythonRelaxDeps = [
+    "sqlalchemy"
+    "cached-property"
+    "tqdm"
+    "fitfile"
+    "tcxfile"
+    "idbutils"
+  ];
+
+  preCheck = ''
+    export HOME=$(mktemp -d)
```

Comment:
Add `writableTmpDirAsHomeHook` to nativeCheckInputs instead.

Reply:
Didn't know that was a thing. Will do.